drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -626,12 +626,13 @@ static void _isr(netdev2_t *netdev)
                  state == AT86RF2XX_STATE_BUSY_TX_ARET) {
             /* check for more pending TX calls and return to idle state if
              * there are none */
+            if (dev->pending_tx > 0) {
+                dev->pending_tx--;
+            }
+
             if (dev->pending_tx == 0) {
```

Comment:
is it guaranteed that `dev->pending_tx > 0` when entering this code path? I could also put an assert there, whadayasay?